zeebe-workflow-manager/src/main/kotlin/workflowmanager/Main.kt

```diff
@@ -0,0 +1,16 @@
+package workflowmanager
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan
+import org.springframework.boot.runApplication
+import org.springframework.scheduling.annotation.EnableScheduling
+
+@SpringBootApplication
+@ConfigurationPropertiesScan("workflowmanager")
+@EnableScheduling
+open class Main
```

Comment:
Maybe rename this main class as e.g. WorkflowManagerApplication or something else?